web-client/src/client/account.rs

```diff
@@ -139,11 +133,20 @@ impl PlainAccount {
 #[derive(serde::Serialize, Tsify)]
 #[serde(rename_all = "camelCase")]
 pub struct PlainStaker {
-    /// The staker's balance.
+    /// The staker's active balance.
     balance: u64,
     /// The address of the validator for which the staker is delegating its stake for. If it is not
     /// delegating to any validator, this will be set to None.
     delegation: Option<String>,
+    /// The staker's inactive balance. Only released inactive balance can be withdrawn from the staking contract.
+    /// Stake can only be re-delegated if the whole balance of the staker is inactive and released
+    /// (or if there was no prior delegation). For inactive balance to be released, the maximum of
+    /// the `inactive_release` and the validator's `jail_release` must have passed.
+    inactive_balance: Coin,
```

Comment:
This is a `PlainStaker` object, and `Coin` does not translate cleanly to a number. Please set this as `u64` like the `balance` field.

Reply:
What does `Coin` translate to in JSON?

Reply:
Didn't #1681 change the JSON serialization of coin to just a simple number?

Reply:
Yea, looks like it: https://github.com/nimiq/core-rs-albatross/pull/1681/commits/a666be6d6deb6900660be084f44960fc6d563fc8#diff-ac8e9d0ded515f48f601114227b21f3b7e850e8a850a7fdc8aad2103f8c8609e.

Reply:
Well, then not JSON, but to WASM. Because what I get now when compiling to WASM is these types:
```
export interface PlainStaker {
    balance: number;
    delegation: string | undefined;
    inactiveBalance: Coin;
    inactiveRelease: number | undefined;
}
```
where `Coin` is not defined and thus a type error.